### PR TITLE
Fix potential duplication of Subscriptions bug

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Move the avatarUrl from `where` to `default`

**Why**
During my local development, I from times to times find out there're duplicate Subscriptions record. I am not 100% sure why, but this `avatarUrl` looks really suspicious. 

**Added feature flags**
N/A

**Affected issues**  
N/A

**How has this been tested?**  
Local
AvatarUrl is available after change

image

**Whats Next?**

### CODE REVIEW

File: src/models/subscription.ts [hunk 1]
```diff
@@ -198,10 +198,10 @@ export class Subscription extends Model {
 				gitHubInstallationId: payload.installationId,
 				jiraHost: payload.host,
 				jiraClientKey: payload.hashedClientKey,
-				gitHubAppId: payload.gitHubAppId || null,
-				avatarUrl: payload.avatarUrl || null
+				gitHubAppId: payload.gitHubAppId || null
 			},
 			defaults: {
+				avatarUrl: payload.avatarUrl || null,
```
Comment (bgvozdev) [inline src/models/subscription.ts:204]: Do we even use it?

Comment (bgvozdev) [reply src/models/subscription.ts:204]: If not, should we rather drop the column?

**Review by bgvozdev — APPROVED** [review]

File: src/models/subscription.ts (hunk 1 above)
Comment (gxueatlassian) [reply src/models/subscription.ts:204]: It is a new column, added by xxxx team. :) I forgot the name.

Comment (gxueatlassian) [reply src/models/subscription.ts:204]: https://github.com/atlassian/github-for-jira/pull/2250/files#r1260508317

**Pull request merged**